addons/l10n_in/views/report_invoice.xml

```diff
@@ -58,6 +58,18 @@
             </h2>
         </xpath>
 
-    </template>
+        <xpath expr="//div[@id='right-elements']" position="inside">
+            <div class="row">
+                <div class="col-6">
+                    <div t-if="'user_id' and not is_html_empty(user_id.signature)" t-out="user_id.signature"/>
+                </div>
+            </div>
+        </xpath>
 
+        <xpath expr="//div[hasclass('clearfix')]" position="after">
+            <xpath expr="//div[@name='comment']" position="move"/>
+            <xpath expr="//div[@id='payment_communication']" position="move"/>
+        </xpath>
```

Comment:
From the commit message, I understood that we just needed to add the signature.
Why do we need to move this? Is there some side effect to adding the signature?

Reply:
This is based on a feedback from Dipali (dit) ; see the task for more detail :)